stock_generate_putaway_from_inventory/models/stock_location.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2021 Akretion (https://www.akretion.com).
+# @author Pierrick Brun <pierrick.brun@akretion.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class StockLocation(models.Model):
+    _inherit = "stock.location"
+
+    def _is_child(self, location):
```

Comment:
Can we use this https://github.com/OCA/stock-logistics-warehouse/tree/13.0/stock_location_children instead of recursive call ?